src/pages/tasks/TaskAssigneeSelectorModal.tsx

```diff
@@ -76,7 +67,11 @@ function useOptions({reports}) {
             true,
         );
 
-        const headerMessage = OptionsListUtils.getHeaderMessage(recentReports?.length + personalDetails?.length !== 0 || currentUserOption, Boolean(userToInvite), debouncedSearchValue);
+        const headerMessage = OptionsListUtils.getHeaderMessage(
+            recentReports?.length + personalDetails?.length !== 0 || Boolean(currentUserOption),
+            Boolean(userToInvite),
```

Comment:
```suggestion
            !!userToInvite,
```

Reply:
I think Boolean is better since it is straight forward conversion

Reply:
The convention is to use !! across files, I agree with @kubabutkiewicz

Reply:
We don't have such a convention in the contributional guidelines. 

I think using `Boolean` is better as you always understand what is going on and do not need to focus on the quantity of `!`